packages/appstore/src/components/loader/mt5-title-skeleton/mt5TitleSkeleton.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './mt5TitleSkeleton.scss';
+
+const Mt5TitleLSkeleton = () => <div className='skeleton-loader mt5-title-skeleton' />;
```

Comment:
Typo!
update to `MT5TitleSkeleton`

Reply:
```suggestion
const MT5TitleSkeleton = () => <div className='skeleton-loader mt5-title-skeleton' />;
```

Reply:
or please let us know if this is intentional

Reply:
@shahzaib-deriv It was typo.